cfn-resources/cloud-backup-snapshot-export-bucket/test/cfn-test-create-inputs.sh

```diff
@@ -32,59 +33,58 @@ echo "roleName: ${roleName} , policyName: ${policyName}"
 
 awsRoleID=$(aws iam get-role --role-name "${roleName}" | jq --arg roleName "${roleName}" -r '.Role | select(.RoleName==$roleName) |.RoleId')
 if [ -z "$awsRoleID" ]; then
-	awsRoleID=$(aws iam create-role --role-name "${roleName}" --assume-role-policy-document file://"$(dirname "$0")"/add-policy.json | jq --arg roleName "${roleName}" -r '.Role | select(.RoleName==$roleName) |.RoleId')
-	echo -e "No role found, hence creating the role. Created id: ${awsRoleID}\n"
+    echo -e "No role found, hence creating the role. Created id: ${awsRoleID}\n"
 else
-	aws iam delete-role-policy --role-name "${roleName}" --policy-name "${policyName}"
-	aws iam delete-role --role-name "${roleName}"
-	awsRoleID=$(aws iam create-role --role-name "${roleName}" --assume-role-policy-document file://"$(dirname "$0")"/add-policy.json | jq --arg roleName "${roleName}" -r '.Role | select(.RoleName==$roleName) |.RoleId')
-	echo -e "FOUND id: ${awsRoleID}\n"
+    aws iam delete-role-policy --role-name "${roleName}" --policy-name "${policyName}"
+    aws iam delete-role --role-name "${roleName}"
+ awsRoleID=$(aws iam create-role --role-name "${roleName}" --assume-role-policy-document "file://$(dirname "$0")/add-policy.json" | jq --arg roleName "${roleName}" -r '.Role | select(.RoleName==$roleName) |.RoleId')
+    echo -e "FOUND id: ${awsRoleID}\n"
 fi
-echo "--------------------------------AWS Role creation ends ----------------------------"
+echo -e "--------------------------------AWS Role creation ends ----------------------------\n"
 
 #------------ get Role arn-------------------
 awsArn=$(aws iam get-role --role-name "${roleName}" | jq --arg roleName "${roleName}" -r '.Role | select(.RoleName==$roleName) |.Arn')
 
-aws iam put-role-policy --role-name "${roleName}" --policy-name "${policyName}" --policy-document file://"$(dirname "$0")"/policy.json
-echo "--------------------------------attach mongodb  Role to AWS Role ends ----------------------------"
+aws iam put-role-policy   --role-name "${roleName}"   --policy-name "${policyName}"   --policy-document "file://$(dirname "$0")/policy.json"
+echo -e "--------------------------------attach mongodb  Role to AWS Role ends ----------------------------\n"
 
 # shellcheck disable=SC2001
 awsArne=$(echo "${awsArn}" | sed 's/"//g')
 # shellcheck disable=SC2086
-#TODO Needs change to while loop using get operation
+# TODO Needs change to while loop using get operation
 sleep 30
 
 atlas cloudProviders accessRoles aws authorize "${roleID}" --projectId "${projectId}" --iamAssumedRoleArn "${awsArne}"
-echo "--------------------------------authorize mongodb  Role ends ----------------------------"
+echo -e "--------------------------------authorize mongodb  Role ends ----------------------------\n"
 
 #create the s3 bucket
 
-bucketName="cloud-backup-snapshot-test123-"${awsRegion}
+bucketName="cloud-backup-snapshot-${CFN_TEST_TAG}-${awsRegion}"
 
-aws s3 rb "s3://${bucketName}" --force
+#aws s3 rb s3://${bucketName} --force
```

Comment:
```suggestion
```

Reply:
we are creating the bucket freshly everytime, bucket name is randomized